README.md

```diff
@@ -43,50 +43,33 @@ Simple Django app that displays a list of curated cafes/spaces to meetup and wor
 1. **Terminal**: [iTerm2](https://www.iterm2.com/) (MacOSX), [Terminator](http://gnometerminator.blogspot.co.uk/p/introduction.html) (Linux) or use your preferred one.
 2. **Text Editor**: [Sublime Text](http://www.sublimetext.com/) or you preferred one.
 
-Install virtualenv: pip install virtualenv
-Install virtualenvwrapper: pip install virtualenvwrapper \*You can skip this and use virtualenv installed in a step before directly, virtualenvwrapper allows for nice interfacing with virtualenv
-Source the virtualenvwrapper: source /usr/local/bin/virtualenvwrapper.sh NOTE: To help do this automatically on every new shell you open add the line above to your .bash_profile or .bashrc
-Create a new env for the project: mkvirtualenv pimp
+## Project Setup
 
-## Laptop Setup
-
-1. [Setup](https://github.com/PolyglotDevsLondon/setup/wiki)
-
-## Virtual Enviroment Setup
-
-This allows you to install all the Python dependencies in a "box" so they are not globally installed and clashing with other projects.
-
-1. Install [virtualenv](https://virtualenv.pypa.io/en/stable/):
-   `pip install virtualenv`
-2. Install [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html):
-   `pip install virtualenvwrapper`
-   \*You can skip this and use [virtualenv](https://virtualenv.pypa.io/en/stable/) installed in a step before directly, virtualenvwrapper allows for nice interfacing with virtualenv
-3. Source the `virtualenvwrapper`:
-   `source /usr/local/bin/virtualenvwrapper.sh`
-   **NOTE**: To help do this automatically on every new shell you open add the line above to your `.bash_profile` or `.bashrc`
-4. Create a new env for the project:
-   `mkvirtualenv sharespots`
+1. Clone the repo: `git@github.com:PolyglotDevsLondon/sharespotsgit`
+2. [Download Docker](https://www.docker.com/products/docker-desktop)
+3. From the root of your project run:
+```
+docker-compose -f local.yml build
+```
+4. Then run:
 
-# Project Setup
+```
+docker-compose -f local.yml up
+```
+4. Make sure you dont have any errors :)
 
-2. Clone the repo: `git@github.com:PolyglotDevsLondon/sharespotsgit`
-3. Create a new virtual environment with virtualenvwrapper: `mkvirtualenv -a sharespots sharespots`
-4. `cd` into the `sharespots` folder
-5. Install all the dependencies: `pip install -r requirements.txt`
-6. [Setup Postgres database locally](#database)
-7. Apply the initial database migrations: `python manage.py migrate`
+Included is django, postgres, pgadmin and jupyter notebook run from django_extensions
 
-## Running the project locally
+# Running the project locally
 
-0. If not already active, activate the virtual environment: `workon sharespots`
-1. Run `python manage.py runserver`
-1. Open your browser at http://localhost:8000/
+1. Run `docker-compose -f local.yml run django python manage.py runserver`
```

Comment:
For me, just running docker "up" (line 57) starts the server too, on 0.0.0.0:8000